graphql/database/users/rewardReferringUser.js

```diff
@@ -92,7 +92,7 @@ const rewardReferringUser = async (userContext, userId) => {
       pendingUserReferralImpact,
       pendingUserReferralCount,
     } = userImpactRecord
-    await UserImpactModel.update(userContext, {
+    await UserImpactModel.update(override, {
```

Comment:
daaaang I'm pretty confident this is the problem.  It would be the referrer's context not the referree's contenxt

Reply:
Ahh, nice catch!